OpenRA.Game/Game.cs

```diff
@@ -83,6 +83,8 @@ public static string TimestampedFilename(bool includemilliseconds = false, strin
 
 		static void JoinInner(OrderManager om)
 		{
+			TextNotificationsManager.Clear();
```

Comment:
Perhaps add a comment here. It feels rather out of place here. Perhaps something like this or more indepth

> Refresh TextNotificationsManager before the game starts

fwiw I'm planning to insert MutedPlayers.Clear() into the this method